Sources/SUIComponents/Helpers/Extensions/CGImage+Stripes.swift

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import CoreImage.CIFilterBuiltins
+
+extension CGImage {
+    static func generateStripePattern(colors: (UIColor, UIColor) = (.clear, .black), width: CGFloat = 2, ratio: CGFloat = 1) -> CGImage? {
+        let context = CIContext()
```

Comment:
Is there really no way to do this in SwiftUI without falling back to CoreImage?

Reply:
Not that I could see. Found the solution here

https://stackoverflow.com/questions/63130755/custom-cross-hatched-background-shape-or-view-in-swiftui

Other solutions I found for swiftui only previded the dashes around the edges, not as a shape fill